projects/packages/google-fonts-provider/README.md

```diff
@@ -0,0 +1,20 @@
+# google-fonts-provider
+
+WordPress Webfonts provider for Google Fonts
```

Comment:
Maybe we can expand on this a bit, and mention the work in progress in the Gutenberg repo, and the previous P2 posts like [this one](https://make.wordpress.org/core/2021/09/28/implementing-a-webfonts-api-in-wordpress-core/)?

Reply:
https://github.com/Automattic/jetpack/pull/23045/commits/9341447ae50d280364b3e8f6e05701d1f7405ec2